src/config.ts

```diff
@@ -93,6 +95,7 @@ export const DEFAULT_QUERY_CONFIG: QueryConfig = {
 
   maxBinsList: [5, 10, 20],
   scaleTypes: [ScaleType.LINEAR, ScaleType.LOG],
+  scaleZeros: [true, false],
```

Comment:
Do you ever refer to this?  (I believe you use the default `[true, false]` [here](https://github.com/uwdata/compassql/pull/113/files#diff-a6322b30cb17d35f81779ff7980ea4c3R109))